packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -122,8 +123,11 @@ const CompleteResetPasswordContainer = ({
     if (accountResetData.verified) {
       // For verified users with OAuth integration, navigate to confirmation page then to the relying party
       if (isOAuthIntegration(integration) && !integration.isSync()) {
-        const sensitiveData = { ...accountResetData, email };
-        sensitiveDataClient.setData('accountResetData', sensitiveData);
+        const sensitiveData = { ...accountResetData };
```

Comment:
Should this be more specific and only set the `unwrapBKey` and `keyFetchToken` in the sensitive data client, since those are the only allowed keys?

Reply:
Oh yes, we can now since we took everything else out from it in the previous revision and I forgot to update this line.

Side-note: I learnt that even with the type system, you can debug this line and see that everything inside `accountResetData` still goes into the `SensitiveDataClient`. Typescript makes no sense to me. 😄